CMakeLists.txt

```diff
@@ -52,6 +52,10 @@ add_library(cpu_features
   src/string_view.c
 )
 
+include(CheckIncludeFiles)
+check_include_files(sys/auxv.h HAVE_SYS_AUXV_H)
```

Comment:
note: some are also checking that getauxval is really present: 
check_function_exists(getauxval HAVE_GETAUXVAL)